plan/plan.go

```diff
@@ -168,23 +200,27 @@ type BlessProvider struct {
 	AWSProfile        *string  `yaml:"aws_profile,omitempty"`
 	AWSRegion         string   `yaml:"aws_region,omitempty"`
 	RoleArn           *string  `yaml:"role_arn,omitempty"`
-	Version           *string  `yaml:"version,omitempty"`
 }
 
 type HerokuProvider struct {
-	Version *string `yaml:"version,omitempty"`
 }
 
 type DatadogProvider struct {
-	Version *string `yaml:"version,omitempty"`
+}
+
+type SentryProvider struct {
+	Enabled bool
+	BaseURL *string `yaml:"base_url,omitempty"`
 }
 
 type TfeProvider struct {
 	Enabled  bool    `yaml:"enabled,omitempty"`
-	Version  *string `yaml:"version,omitempty"`
 	Hostname *string `yaml:"hostname,omitempty"`
 }
 
+type KubernetesProvider struct {
```

Comment:
above looked like this provider can have a hostname configuration, should it be here too?

Reply:
I was intending to only do enough to support the required_providers part of setting up the kubernetes provider, continuing to rely on hand-writing `provider` blocks for kubernetes for now. I don't see a hostname config, there might have been one in an earlier version of this PR.